src/Polly.Net35/CircuitBreaker/SuccessRatioCircuitBreakerState.cs

```diff
@@ -57,8 +64,12 @@ public void TryBreak(Exception ex)
             {
                 _lastException = ex;
                 _failCount += 1;

```

Comment:
This should happen after the decay. Otherwise the `1` that you added will get decayed by the time since you last updated it.